packages/analytics-browser/src/storage/browser-storage.ts

```diff
@@ -1,7 +1,12 @@
+import { Logger } from '@amplitude/analytics-types';
 import { Storage as AmplitudeStorage } from '@amplitude/analytics-types';
 
 export class BrowserStorage<T> implements AmplitudeStorage<T> {
-  constructor(private storage?: Storage) {}
+  logger?: Logger;
+
+  constructor(private storage?: Storage, logger?: Logger) {
```

Comment:
do we need this?

Reply:
I was thinking about extend this to session storage as well. But it's necessary for this fix. Deleted it.